.github/workflows/ci.yml

```diff
@@ -46,6 +46,7 @@ jobs:
           --exclude '[^\w]todo[^\w]?'
           --exclude 'https://(www\.|old\.)?reddit\.com'
           --exclude 'https://www.patreon.com'
+          --exclude 'https://gamedev.social'
```

Comment:
Probably not to keep ultimately

Reply:
Yep, I'll remove it again once they get their stuff together 😅